packages/components/src/components/InfoItem/utils.ts

```diff
@@ -0,0 +1,49 @@
+import { spacings, SpacingValues, TypographyStyle } from '@trezor/theme';
+
+import { FlexAlignItems } from '../Flex/Flex';
+import { IconSize } from '../Icon/Icon';
+import { InfoItemVerticalAlignment } from './types';
+
+export const mapVerticalAlignmentToAlignItems = (
+    verticalAlignment: InfoItemVerticalAlignment,
+): FlexAlignItems => {
+    const alignItemsMap: Record<InfoItemVerticalAlignment, FlexAlignItems> = {
+        top: 'flex-start',
+        center: 'center',
+        bottom: 'flex-end',
+    };
+
+    return alignItemsMap[verticalAlignment];
+};
+
+export const mapTypographyStyleToIconSize = (
+    typographyStyle: TypographyStyle,
+): IconSize | number => {
+    const iconSizeMap: Record<TypographyStyle, IconSize | number> = {
```

Comment:
This leads me to that we might should unify somehow `small | medium | large` with `xs, sm, lg, xl, xxl` etc. (as we have in spacings). WDYT? (definitely out of scope, but just thing to think about for the future)